routes/freelance/router.js

```diff
@@ -26,8 +26,7 @@ router.get('/new', function(req, res, next) {
           title: "JobAdvisor - Create Freelancer Profile" ,
           logged: true,
           username: user.username,
-          userFreelancer: user.freelancer,
-          claiming: user.claiming
+          userFreelancer: user.freelancer[0]
```

Comment:
Kk, we are doing that page on `feature/user-page`. That page will be public, every user can see it, it's just a list of the freelancers owned by that user, each linking to the actual freelancer profile. The links to edit will be on the freelancer profile (if the user is the owner).